src/store/publicChannels/publicChannels.slice.ts

```diff
@@ -3,54 +3,78 @@ import { createSlice, EntityState, PayloadAction } from '@reduxjs/toolkit';
 import { StoreKeys } from '../store.keys';
 
 import {
-  publicChannelMessagesAdapter,
+  publicChannelMessagesIdsAdapter,
   publicChannelsAdapter,
 } from './publicChannels.adapter';
-import { IMessage, IChannelInfo } from './publicChannels.types';
+import { IChannelInfo, IMessage } from './publicChannels.types';
 
 export class PublicChannelsState {
-  public currentChannelAddress: string = '';
   public channels: EntityState<IChannelInfo> =
     publicChannelsAdapter.getInitialState();
-  public messages: EntityState<IMessage> =
-    publicChannelMessagesAdapter.getInitialState();
+
+  public currentChannelMessagesIds: EntityState<string> =
```

Comment:
Well, I know that in your app you only have one channel, but what is the plan when you have more than one? 
I think we need a structure like:

```
{ [channelId: string]: { ids: string[], data: [id: string]: IMessage } }
```
This way you track state of multiple channels.

"currentChannelMessagesIds" becomes a selector, that draws from currentChannelId key